overlord/servicestate/quota_handlers.go

```diff
@@ -24,6 +24,9 @@ import (
 	"sort"
 	"time"
 
+	tomb "gopkg.in/tomb.v2"
+
+	"github.com/snapcore/snapd/gadget/quantity"
```

Comment:
May be worth considering moving the quantity package around to some better location in the future.